### DISCUSSION TITLE
Example of binding to a PasswordBox in dialog

### DISCUSSION THREAD

**Discussion by shaggygi** [PrismLibrary/Prism#2668] [WPF] [answered]
Scratching my head on how to get around the security issue with binding to a PasswordBox Password.  I've seen explanations regarding issue via searches, but is more specific and not related to Prism.  Was hoping the community might could help.

Anyone have examples that could get me going using with Prism?  Feedback is appreciative.

**Comment by brianlagunas** [comment, marked as answer]
A password box isn’t secure. Just bind to the plain text directly via an attached property.

**Reply by shaggygi** [reply]
Gotcha.  That did the trick.  Thanks
